### PR TITLE
Adding Flovatar transactions

### PR DESCRIPTION
**Pull request by luca-mayven**
Hi, I'm adding some basic transactions that you can test on flovatar.com

Thanks!

### CODE REVIEW

File: transactions/Flovatar/buyPack.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        "saleAddress",
+        "tokenId",
+        "amount",
+        "signature"
+    ],
+    "messages": {
+        "en": "Buy a Flovatar Pack for {amount} FUSD"
+    },
+    "balances": {
+        "bidAmount": "FUSD"
```
Comment (boczeratul) [inline transactions/Flovatar/buyPack.json:12]: `bidAmount` needs to be consistent with argument `amount`

File: transactions/Flovatar/buyFlovatarComponent.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "arguments": [
+        "saleAddress",
+        "tokenId",
+        "amount"
+    ],
+    "messages": {
+        "en": "Buy a Flovatar Component for {amount} FUSD from {saleAddress}"
+    },
+    "balances": {
+        "bidAmount": "FUSD"
```
Comment (boczeratul) [inline transactions/Flovatar/buyFlovatarComponent.json:11]: `bidAmount` needs to be consistent with argument `amount`

File: transactions/Flovatar/buyFlovatar.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    "arguments": [
+        "saleAddress",
+        "tokenId",
+        "amount"
+    ],
+    "messages": {
+        "en": "Buy a Flovatar for {amount} FUSD from {saleAddress}"
+    },
+    "balances": {
+        "bidAmount": "FUSD"
```
Comment (boczeratul) [inline transactions/Flovatar/buyFlovatar.json:11]: `bidAmount` needs to be consistent with argument `amount`

**Comment by luca-mayven** [comment]
Sorry for missing that and thanks for letting me know!
I just updated the fields.
Thanks

**Pull request merged**